### PR TITLE
Add README_CN.md and fix typos in README.md

### PR DESCRIPTION
**Pull request by tinafangkunding**
## What has been implemented?

Add README_CN.md 

## Todos:

* [ - ] Write tests
* [ √ ] Write / update documentation
* [ √ ] Run Prettier

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -562,7 +567,7 @@ During development, it's super helpful to keep the terminal running with `server
 
 #### Use Debug Statements and the `--debug` Flag
 
-The Serverless Components CLI experience is intentionally minimal. But if you ever want to see what Components are doing behnd the scenes, use the `--debug` flag when you run a Component. You'll see output like this:
+The Serverless Components CLI experience is intentionally minimal. But if you ever want to see what Components are doing behand the scenes, use the `--debug` flag when you run a Component. You'll see output like this:
```
Comment (medikoo) [inline README.md]: Small typo, should be: _behind_

Comment (tinafangkunding) [reply README.md]: Sorry, I'll fix it.

**Comment by tinafangkunding** [comment]
Hi @medikoo, can you please help to review and merge the pr?  In the serverless/serverless README_CN.md, there is an url link to this doc, It's better to merge both of them, thanks a lot!

**Comment by tinafangkunding** [comment]
CC @pmuens, could you please help us to review and merge this pr? Thanks!

**Comment by tinafangkunding** [comment]
Thanks a lot!

**Pull request merged**